EventsExpress/ClientApp/src/components/admin/admin.js

```diff
@@ -1,16 +1,55 @@
 import React, { Component } from 'react';
+import { Route } from "react-router-dom";
 import './admin.css';
+import { NavItem } from '../NavItem/NavItem';
+import Category from '../category/categories';
+import UserPWrapper from '../../containers/UsersWrapper';
+import UnitOfMeasuring from '../unitOfMeasuring/unitsOfMeasuring';
 
+export default class Admin extends Component {
 
-export default class Admin extends Component{
-    
-    render(){
-     
-    return(
-        <div>
-        
-        </div>
-        
-    );
+    render() {
+
+        return (
+            <>
+                <div className="admin-panel row">
+                    <h3>Admin Panel</h3>
+                </div>
+                <div className="row h-100">
+                    <div className="admin-sidebar col-sm-2">
+                        <ul className="list-unstyled">
+                            <nav id="sub-nav">
+                                <div>
+                                    <NavItem                                                                               
+                                        to={'/admin/categories/'}
+                                        icon={'fa fa-hashtag'}
+                                        text={"Categories"}
+                                    />
+                                </div>
+                                <div>
+                                    <NavItem
+                                        to={'/admin/unitsOfMeasuring/'}
+                                        icon={'fa fa-plus'}
+                                        text={"UnitsOfMeasuring"}
+                                    />
+                                </div>
+                                <div>
+                                    <NavItem
+                                        to={'/admin/users?page=1'}
+                                        icon={'fa fa-users'}
+                                        text={"Users"}
+                                    />
+                                </div>
+                            </nav>
+                        </ul>
+                    </div>
+                    <div className="col-sm-9 offset-sm-1">                        
+                            <Route path="/admin/categories/" component={Category} />
+                            <Route path='/admin/unitsOfMeasuring' component={UnitOfMeasuring} />
+                            <Route path="/admin/users" component={UserPWrapper} />                    
```

Comment:
Shouldn't we add these under the main **admin** route in `app.js` and here just render `{children}`?